src/views/tickets/BaseTicketList.vue

```diff
@@ -38,6 +39,7 @@ export default {
     const vm = this
     return {
       loading: true,
+      getDrawerTitle: () => '',
       createDrawer: () => import('@/views/tickets/RequestAssetPerm/CreateUpdate'),
       detailDrawer: null,
       ticketTableConfig: {
```

Comment:
I'm sorry, but I cannot see the specific parts of the code you want to analyze. You need to send all the code for accurate evaluation and analysis.